tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs

```diff
@@ -284,6 +286,8 @@ public void GetPlugins()
 
         ProviderNames = new ObservableCollection<string>(_providers.GetAllProviderNames());
         TelemetryFactory.Get<ITelemetry>().Log("RepoTool_SearchForProviders_Event", LogLevel.Critical, new ProviderEvent(ProviderNames.Count), _activityId);
+
+        IsAccountButtonEnabled = plugins.Any();

```

Comment:
Is IsAccountButtonEnabled automatically associated with _isAccountButtonEnabled?

Reply:
Yes.  The connection is made in the back end with the community toolkit.

Reply:
It's the magic of marking it [ObservableProperty]